### DISCUSSION TITLE
Integration with or implementation of Tasks plugin from Obsidian

### DISCUSSION THREAD

**Discussion by zan-preston** [epwalsh/obsidian.nvim#575] [Ideas]
It would be nice to be able to have similar functionality to the Tasks plugin in Obsidian here. obsidian://show-plugin?id=obsidian-tasks-plugin

My idea for how it would work in my head would be something like having the code block

<pre>
```task
not done
```
</pre>

and then when not in the block (similar to how conceal works) have tasks populated.

An alternative would potentially be some sort of Telescope picker or UI like Neogit/Fugit2 but for tasks that would allow you to just go in and query/view tasks across your vault and then interact with them. This might be the better ux design for a neovim workflow.

**Comment by jkmacc-LANL** [comment]
I've tried this with Telescope grep on `- \[ \]` to find incomplete tasks, but it doesn't do any of the nice sorting or filtering that the Obsidian Tasks plugin does.
